lib/Rx/Observable.php

```diff
@@ -534,6 +707,10 @@ public function distinctKey(callable $keySelector, callable $comparer = null)
      *
      * @param callable $comparer
      * @return \Rx\Observable\AnonymousObservable
+     *
+     * @demo distinct/distinctUntilChanged.php
+     * @operator
+     * @reactivex distinct
      */
     public function distinctUntilChanged(callable $comparer = null)
```

Comment:
"A variant of distinct that only compares emitted items from the source Observable against their immediate predecessors in order to determine whether or not they are distinct."